src/ui/views/History/style.less

```diff
@@ -0,0 +1,148 @@
+@import '../../style/var.less';
+
+.txs-history {
+  .page();
+  display: flex;
+  flex-direction: column;
+  padding: 0 20px 20px;
+  font-size: 12px;
+  line-height: 14px;
+  height: 100vh;
+  overflow: auto;
+
+  // padding-top: 72px;
+  // .page-header {
+  //   position: fixed;
+  //   top: 0;
+  //   right: 20px;
+  //   left: 20px;
+  //   margin-bottom: 0;
+  //   padding-bottom: 20px;
+  //   z-index: 10;
+  //   background-color: #f0f2f5;
+  // }
+  &-card {
+    position: relative;
+    background: #ffffff;
+    border-radius: 6px;
+    padding: 0 8px;
```

Comment:
![image](https://user-images.githubusercontent.com/86296331/149260918-3fb1739e-9188-4e21-92ae-6be3059eb832.png)
should be 12px

Reply:
![image](https://user-images.githubusercontent.com/86296331/149261851-90c57fee-d149-4b83-9586-5e5d233abf95.png)
looks like `text-overflow` is necessary